internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -284,6 +285,66 @@ func (opts *Opts) askClusterOptions() error {
 	return nil
 }
 
+func (opts *Opts) defaultRegions() []string {
+	cloudProviders, err := opts.store.CloudProviderRegions(opts.ConfigProjectID(), opts.Provider, tier, false)
+
+	if err != nil {
+		return DefaultRegions[strings.ToUpper(opts.Provider)]
+	}
+
+	if err := validateDefaultRegions(cloudProviders); err != nil {
+		return DefaultRegions[strings.ToUpper(opts.Provider)]
+	}
+
+	availableRegions := cloudProviders.Results[0].InstanceSizes[0].AvailableRegions
+
+	defaultRegions := make([]string, len(availableRegions))
+
+	// the most popular region must be the first in the list
+	popularRegion := ""
+	for _, v := range availableRegions {
+		if v.Default {
+			popularRegion = v.Name
+			break
+		}
+	}
+
+	defaultRegions[0] = popularRegion
+	i := 1
+
+	for _, v := range availableRegions {
+		if v.Name != popularRegion {
+			defaultRegions[i] = v.Name
+			i++
+		}
+	}
+
+	return defaultRegions
```

Comment:
[nit] if you find the index of the default you could move it in place to be the first, this is a bit more expected in go

Another option instead of using your own index is using `append` to the array

Finally please consider creating a separate function to find default